resolwe/test_helpers/test_runner.py

```diff
@@ -515,17 +543,41 @@ def resolve_process_tags(self, files):
         for proc_path in processes_paths:
             process_schemas.extend(self.find_schemas(proc_path))
 
+        # Switch to source branch and get all the schemas from there as well, since
```

Comment:
Same as above, can you wrap the comments at the same wrap limit as the surrounding code?